app/views/docs/authentication-server.phtml

```diff
@@ -0,0 +1,638 @@
+<p>
+    You can exand Appwrite's APIs by building backend apps using <a href="/docs/command-line">Server SDKs</a>. Appwrite enables you to authenticate your Web, Flutter, Android, or Apple application directly with your backend apps using JWT authentication.
```

Comment:
```suggestion
    You can extend Appwrite's APIs by building backend apps using <a href="/docs/command-line">Server SDKs</a>. Appwrite enables you to authenticate your Web, Flutter, Android, or Apple application directly with your backend apps using JWT authentication.
```

The href is pointing to the CLI. It needs to be updated

Reply:
> Appwrite enables you to authenticate your Web, Flutter, Android, or Apple application directly with your backend apps using JWT authentication.

What is the significance of this sentence? It is not entirely true as well. 

Developers can authenticate their server sdks either using an API key with a project scope or a JWT if they would like to authenticate in the scope of a individual user.